releases/0.2.0.ramstage.md

```diff
@@ -0,0 +1,207 @@
+# Dasharo Trustworthy Computing
+
+[https://dasharo.com](https://dasharo.com)
+
+Talos II support in coreboot
+
+## v0.2.0
+
+### Release Changelog
+\
+**Added:**
+* Added building of skiboot image
```

Comment:
@IgorBagnucki Can we try to elaborate more on those changes? E.g. add second level of bullets and explain each point in more details, maybe with some references as well